README.md

```diff
@@ -224,6 +225,15 @@ If your existing resource is in one of those regions, then you can re-use it by
 You can also use existing Azure AI Document Intelligence and Storage Accounts. See `./infra/main.parameters.json` for list of environment variables to pass to `azd env set` to configure those existing resources.
 
 
+### Using local resources
```

Comment:
I think this should be included to have different local options other than the parsers in the future.

Reply:
Sure! I can add local OpenAI to it later.

Reply:
* Err, local LLMs with OpenAI compatible endpoints.

Reply:
Yep, I saw you taking about this in the live streams multiple times